wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py

```diff
@@ -204,7 +207,7 @@ def __init__(self, aff_obj):
 
         if aff_obj.OrganizationObj is not None:
             self.Organization = aff_obj.OrganizationObj.OrganizationName
-            self.OrganizationCode = aff_obj.OrganizationObj.OrganizationCode
+            self.SourceCode = aff_obj.OrganizationObj.OrganizationCode
             self.SourceDescription = aff_obj.OrganizationObj.OrganizationDescription
```

Comment:
> `self.SourceCode = aff_obj.OrganizationObj.OrganizationCode`

This is wrong, at least according to the interpretation I laid out in https://github.com/ODM2/WOFpy/issues/162#issue-248940883
There, I suggested it should be:
`self.SourceCode = aff_obj.AffiliationID`
or even more explicitly:
`self.SourceCode = self.SourceID`

Thoughts?